packages/auth/src/resourceSet/routes.ts

```diff
@@ -0,0 +1,41 @@
+import { AppContext } from '../app'
+import { BaseService } from '../shared/baseService'
+import { ResourceSetService } from './service'
+
+interface ServiceDependencies extends BaseService {
+  resourceSetService: ResourceSetService
+}
+
+export interface ResourceSetRoutes {
+  create(ctx: AppContext): Promise<void>
+}
+
+export function createResourceSetRoutes({
+  logger,
+  resourceSetService
+}: ServiceDependencies): ResourceSetRoutes {
+  const log = logger.child({
+    service: 'ResourceSetRoute'
+  })
+
+  const deps = {
+    logger: log,
+    resourceSetService
+  }
+
+  return {
+    create: (ctx: AppContext) => createResourceSet(deps, ctx)
+  }
+}
+
+async function createResourceSet(
+  deps: ServiceDependencies,
+  ctx: AppContext
+): Promise<void> {
+  const { body } = ctx.request
+  const resourceSet = await deps.resourceSetService.create(body)
```

Comment:
I'm confused about where the `key` comes from
>The RS MUST identify itself with its own key and sign the request.

but the immediate subsequent example doesn't include the `key` in the request body.
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.4